corehq/apps/registration/tasks.py

```diff
@@ -57,7 +57,7 @@ def activation_24hr_reminder_email():
 
 @task(serializer='pickle', queue="email_queue")
 def send_domain_registration_email(recipient, domain_name, guid, full_name, first_name):
-    registration_link = 'http://' + DNS_name + reverse('registration_confirm_domain') + guid + '/'
+    registration_link = 'http://' + get_site_domain() + reverse('registration_confirm_domain') + guid + '/'
```

Comment:
@orangejenny i think there might be a problem here if the `http://` is left in?

Reply:
What do you mean? All I'm intending to do is restore the old code: https://github.com/dimagi/commcare-hq/pull/26935/files#diff-0890a99a4eb7f4a7eace79122bf9574cL62-L63

Reply:
@orangejenny i read the commit incorrectly...all good